pkgs/tools/misc/vmtouch/default.nix

```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "vmtouch";
-  version = "git-20150310";
+  version = "1.0.2";
```

Comment:
I can do that if that's the consensus, but it seems strange to me to keep `git-` in the name indefinitely if it's using a released version from now on.